graylog2-server/src/main/java/org/graylog2/configuration/ElasticsearchConfiguration.java

```diff
@@ -132,6 +134,9 @@ public class ElasticsearchConfiguration {
     @Parameter(value = TIME_SIZE_OPTIMIZING_RETENTION_MAX_LIFETIME)
     private Period timeSizeOptimizingRotationMaxLifeTime = TimeBasedSizeOptimizingStrategyConfig.DEFAULT_LIFETIME_MAX;
 
+    @Parameter(value = TIME_SIZE_OPTIMIZING_RETENTION_FIXED_LEEWAY)
+    private Period timeSizeOptimizingRotationFixedLeeway = Period.days(1);
```

Comment:
for on-prem we don't want this to be on by default.
can we set the default to null? and only validate it if it's set?